CHANGELOG.md

```diff
@@ -0,0 +1,102 @@
+# Changelog
+
+## Version 1.20 (Jul 29, 2019)
+
+-   [JENKINS-33217](https://issues.jenkins-ci.org/browse/JENKINS-33217) Log a warning when additional parameters are passed into \`DescribableModel\` objects, such as Pipeline steps, since these parameters are currently being ignored.
+-   Internal - Update parent pom ([PR-46](https://github.com/jenkinsci/structs-plugin/pull/46){.external-link}) and fix unit tests ([PR-50](https://github.com/jenkinsci/structs-plugin/pull/50){.external-link})
```

Comment:
You need to strip all the `{.external-link}` tags. The wiki markdown converter unfortunately leaves all of those in place.